src/uwtools/cli/experiment_manager.py

```diff
@@ -50,13 +52,6 @@ def parse_args(args: List[str]) -> Namespace:
         type=cli_helpers.path_if_it_exists,
     )
     optional = parser.add_argument_group("optional arguments")
-    optional.add_argument(
-        "-l",
-        "--log-file",
-        default="/dev/null",
-        help="Path to a file to log to",
-        metavar="FILE",
-    )
```

Comment:
Few tools that I'm aware of offer an option to redirect non-data (i.e. for humans) output.

- Like the version of `uw` in this PR, `conda create` offers `--quiet` and `--verbose` options, but no redirection option.
- `npm` has a `--verbose` switch and, at least in some modes, `--silent`, but no redirection option that I can find.
- `git` has `--quiet` and `--verbose` but no redirection.

This follows Unix tradition of keeping messages and data on separate streams and providing tools to redirect from the console if desired. For example, with `git` the error output from specifying a bad mode can be directed to a file, including to `/dev/null` to suppress it:
```
% git asdf # normal behavior
git: 'asdf' is not a git command. See 'git --help'.

The most similar command is
	add
```
```
% git asdf 2>err # send to file 'err'
% cat err
git: 'asdf' is not a git command. See 'git --help'.

The most similar command is
	add
```
```
% git asdf 2>/dev/null # suppress stderr
% 
```
So, barring unforeseen circumstances, I'd like to suggest that YAGNI. It will be easy to add this option via the modal CLI in one place for all modes, if we find we need it.
 
(Full disclosure: `curl`, like the proposed `uw`, streams data to `stdout` by default (and offers a `-o` switch to direct data to a file, as I believe `uw` should, too), as well as `--silent` and `--verbose` switches. It does also have a `--stderr` option to redirect its own human-readable output without using shell redirection.)